sycl/test-e2e/bindless_images/array/read_write_1d_subregion.cpp

```diff
@@ -1,7 +1,7 @@
-// REQUIRES: cuda
+// REQUIRES: aspect-ext_oneapi_image_array
 
-// RUN: %{build} -o %t.out
-// RUN: %{run} %t.out
+// RUN: %if !any-device-is-hip %{ %{build} -o %t.out %}
```

Comment:
Is this not the same as doing `// UNSUPPORTED: HIP`? If so, would that style not be better?

Reply:
Yeah it absolutely is. That's a weird leftover I didn't spot. Should be changed to `// UNSUPPORTED: HIP` and will do. (Think I did that before knowing I had to do `UNSUPPORTED-INTEDED` for unsupported targets :sweat_smile:)

Reply:
In fact, this is already handled by the required aspect at the start of the test. UR returns false for HIP here so that won't run at all. Just going to remove this conditional leftover from the RUN lines.